src/components/Country.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { getCountry } from '../redux/country/country.js';
+import CountryItem from './CountryItem.js';
+import Header from './Header.js';
+
+const Country = () => {
```

Comment:
- [ ] **[OPTIONAL]** The image below shows your homepage image on the desktop. While this project is intended as a mobile application, it would be nice to reduce the height of your homepage image on the desktop for a better image resolution and User Experience (UX).

![musta-home](https://user-images.githubusercontent.com/36542986/142446656-9baa8164-6371-413c-a934-99e3b044663d.jpg)